interfaces/builtin/common.go

```diff
@@ -142,3 +145,13 @@ func (iface *commonInterface) SecCompConnectedPlug(spec *seccomp.Specification,
 	}
 	return nil
 }
+
+func (iface *commonInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.Plug, plugAttrs map[string]interface{}, slot *interfaces.Slot, slotAttrs map[string]interface{}) error {
```

Comment:
Thank you for doing this. I wanted to add something like this for a *long* time lately